File: WordPress/src/test/java/org/wordpress/android/models/SiteSettingsModelTest.java
package org.wordpress.android.models;

import org.junit.Test;

import nl.jqno.equalsverifier.EqualsVerifier;
import nl.jqno.equalsverifier.Warning;

public class SiteSettingsModelTest {
    @Test
    public void equalsContract() {
        EqualsVerifier
                .forClass(SiteSettingsModel.class)
                .suppress(Warning.NONFINAL_FIELDS)
                .verify();
    }
}

File: WordPress/src/test/java/org/wordpress/android/models/SiteSettingsModelTest.kt
package org.wordpress.android.models

import org.junit.Assert.assertEquals
import org.junit.Test
import org.wordpress.android.Fixtures.createFakeSiteSettingsModel

class SiteSettingsModelTest {
    @Test
    fun copyFrom() {
        val original = createFakeSiteSettingsModel()
        val copy = SiteSettingsModel()
        copy.copyFrom(original)
        assertEquals(original, copy)
    }
}
